### PR TITLE
Create global-coordinate.js

### PR DESCRIPTION
**Pull request by NOname-awa**

### CODE REVIEW

**Comment by TheShovel** [comment]
When the position of the screen is changed, rotation doesn't work correctly. It shouldn't rotate around the middle of the canvas, it should rotate around the middle of the screen, using it's coordinates.

**Comment by TheShovel** [comment]
[video.webm](https://user-images.githubusercontent.com/68913917/232532062-51260388-909a-489b-b27f-bbbd97fc3f18.webm)

File: extensions/NOname-awa/global-coordinate.js [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+			}
+		}
+		Set_si({ s, screen }) {
+			if (!isNaN(s)) {
+				gs[screen - 1] = s;
+			}
+			else {
+				gs[screen - 1] = 0;
+			}
+			if (gs < 0) {
```
Comment (GarboMuffin) [inline extensions/NOname-awa/global-coordinate.js:455]: This looks wrong
```
Operator '<' cannot be applied to types 'number[]' and 'number'.ts(2365)
```

gs is an array; why is it being compared to a number? are you sure this does what you want?

**Comment by GarboMuffin** [comment]
bumping  per above comment

**Pull request merged**